oauth2client/contrib/django_util/__init__.py

```diff
@@ -106,7 +106,7 @@ def requires_default_scopes(request):
                     http=request.oauth.http,
```

Comment:
This will likely conflict with #546. @waprin do you think he should revert this and let you handle it, or do you think we can deal with the merge conflict gracefully?